configs/detection_fomo_heavy_model.yaml

```diff
@@ -0,0 +1,42 @@
+# Example configuration for training a predefined fomo heavy detection model
+# NOTE: This example downloads pretrained COCO weights and training parameters are already prepared for fine tuning
```

Comment:
MobileNetv2 weights are from ImageNet ([source](https://pytorch.org/vision/main/models/generated/torchvision.models.mobilenet_v2.html#torchvision.models.MobileNet_V2_Weights)) - same for other config